charts/cluster-api-runtime-extensions-nutanix/templates/helm-config.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+data:
+  aws-ccm-config: |
+    defaultChartName: aws-cloud-controller-manager
+    defaultChartVersion: ""
```

Comment:
Do we need to come up with a different structure that also satisfies addons that are k8s version specific?

Reply:
Im open to suggestions!